osf/models/institution.py

```diff
@@ -16,8 +16,8 @@
 from osf.models import base
 from osf.models.contributor import InstitutionalContributor
 from osf.models.mixins import Loggable, GuardianMixin
+from website import mails
 from website import settings as website_settings
-from website.mails import send_mail, INSTITUTION_DEACTIVATION
```

Comment:
This is the fix that makes mocking `send_mail` work in the tests.

Importing `send_mail` from `mails` breaks how mock works when the tests try to mock patch the `send_mail` and assert call count and args. Must import `mails` directly from `website` and then access `send_mail` via mails.send_mail` in the model class in order for the mock to work in unit tests.

Reply:
oh yeah! i forgot about that nuance -- the other option would have been mocking `osf.models.institution.send_mail` instead of `website.mails.send_mail`, so it directly replaces what was imported in the model file -- both good paths